package.json

```diff
@@ -12,6 +12,7 @@
     "lint": "eslint src/** --ext .js --ext .jsx --cache",
     "storedist": "npm run build && git add -f dist",
     "comparedist": "bash ./scripts/test_compare_dist_with_committed_dist.sh",
+    "syncmock": "node src/test/fetchMockData.js --diff",
```

Comment:
Thanks for reviewing! It runs `git diff --quiet` on the file and looks at the exit code. This works because the mockData has been checked in. I followed the same strategy for the screenshot test.